content/courses/connecting-to-offchain-data/oracles.md

```diff
@@ -1360,14 +1392,16 @@ describe("burry-escrow", () => {
         .signers([payer])
         .rpc();
 
-      await provider.connection.confirmTransaction(tx, "confirmed");
-      console.log("Your transaction signature", tx);
-    } catch (e) {
-      // verify tx returns expected error
+      await provider.connection.confirmTransaction(
+        transactionSignature,
+        "confirmed",
+      );
+      console.log("Your transaction signature:", transactionSignature);
+    } catch (error) {
       didFail = true;
-      console.log(e.error.errorMessage);
+      console.error(error.errorMessage);
       assert(
-        e.error.errorMessage ==
+        error.errorMessage ===
```

Comment:
`error.errorMessage` seems bad:

 - Errors have `message` not `errorMessage`.See. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/message
 - If this *does* have an `errorMessage` it isn't an error.
 
 Check the code and modify it accordingly.